test/integration/modules/edgecore/eventbus/eventbus_test.go

```diff
@@ -19,12 +19,10 @@ package integration
 import (
 	"database/sql"
 	"encoding/json"
-	"os"
-	"path/filepath"
-	"time"
-
 	"github.com/kubeedge/kubeedge/pkg/devicetwin/dtcommon"
 	"github.com/kubeedge/kubeedge/test/integration/utils/common"
+	"os"
```

Comment:
Put these imported package in suitable place?

Reply:
@m1093782566 , OK..